Source/Core/DolphinWX/ISOFile.cpp

```diff
@@ -129,7 +133,11 @@ GameListItem::GameListItem(const std::string& _rFileName)
 		ini.GetIfExists("EmuState", "EmulationIssues", &m_issues);
 	}
 
-	if (!m_pImage.empty())
+	if (IsElfOrDol())
+	{
+		m_Bitmap.LoadFile(StrToWxStr(File::GetThemeDir(SConfig::GetInstance().theme_name)) + "filebanner.png", wxBITMAP_TYPE_PNG);
```

Comment:
I'd agree tho. Instead of exposing ```IsElfOrDol``` and deciding in the UI, it would probably be nicer to simply return appropriate values within the given methods (such as ```GetCountry```, ```GetEmuState``` and even ```GetFileSize```). Especially since that way, other existing code wouldn't break if it called either of those on an ELF/DOL without checking for ```IsElfOrDol``` first.